### PR TITLE
clair 4.0.1

### PR DESCRIPTION
**Pull request by chenrui333**
[`action-homebrew-bump-formula`](https://github.com/dawidd6/action-homebrew-bump-formula)

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by dtrodrigues — COMMENTED** [review]
Can `test_resource` be removed per its comment?

**Comment by carlocab** [comment]
```
❯ brew unbottled --tag=arm64_big_sur clair
==> Populating dependency tree...
==> :arm64_big_sur bottle status
clair: unbottled deps: rpm
```

**Comment by chenrui333** [comment]
> Can `test_resource` be removed per its comment?

absolutely! 👍

File: Formula/clair.rb [hunk 1]
```diff
@@ -20,25 +20,18 @@ class Clair < Formula
...
 
     system "go", "build", *std_go_args, "-ldflags", ldflags, "./cmd/clair"
-    (etc/"clair").install resource("test_resource")
+    (etc/"clair").install "config.yaml.sample" => "config.yaml"
   end
 
   test do
-    cp etc/"clair/config.yaml.sample", testpath
+    cp etc/"clair/config.yaml", testpath
     output = shell_output("#{bin}/clair -conf #{testpath}/config.yaml.sample -mode combo 2>&1", 1)
```
Comment (carlocab) [inline Formula/clair.rb:35]:
```suggestion
    output = shell_output("#{bin}/clair -conf #{testpath}/config.yaml -mode combo 2>&1", 1)
```
?

Comment (chenrui333) [reply Formula/clair.rb]: missed it 😅

**Comment by carlocab** [comment]
```
An exception occurred within a child process:
Error: clair: failed
  Test::Unit::AssertionFailedError: </initialized\ failed:\ failed\ to\ initialize\ libindex:\ failed\ to\ create\ ConnPool/> was expected to be =~
<"2021/02/16 01:07:22 failed to decode yaml config: yaml: line 30: found character that cannot start any token\n">.
```
I guess the resource can't be removed yet.

**Comment by chenrui333** [comment]
The change is not in 4.0.1 release.  :(

Let's how it goes for the next release.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**